src/textual/app.py

```diff
@@ -427,6 +429,11 @@ def __init__(
         self.set_class(self.dark, "-dark-mode")
         self.set_class(not self.dark, "-light-mode")
 
+    @property
+    def workers(self) -> WorkerManager:
+        """A worker manager."""
+        return self._workers
```

Comment:
I think "workers" makes us think this will give us the workers being manager, not the worker manager itself.
I suggest `workers -> worker_manager` and `_workers -> _worker_manager`.

Reply:
The manager does contain the workers, so it is in a sense the workers being managed.